APMrover2/GCS_Mavlink.cpp

```diff
@@ -1091,19 +1091,22 @@ void GCS_MAVLINK_Rover::handleMessage(mavlink_message_t* msg)
 
             float target_speed = 0.0f;
             float target_yaw_cd = 0.0f;
+            float target_dir = 1.0f;
 
             // consume velocity and convert to target speed and heading
             if (!vel_ignore) {
                 // convert vector length into a speed
                 target_speed = constrain_float(safe_sqrt(sq(packet.vx) + sq(packet.vy)), -rover.g.speed_cruise, rover.g.speed_cruise);
                 // convert vector direction to target yaw
                 target_yaw_cd = degrees(atan2f(packet.vy, packet.vx)) * 100.0f;
+                if (is_negative(target_yaw_cd)) {
```

Comment:
I think the target_yaw_cd will be negative if the vector is left of north (if in earth frame) or left of the vehicle's heading (if in body frame).  So won't this make the vehicle backup if someone requests it head west?